modules/storage-page-memory/src/main/java/org/apache/ignite/internal/storage/pagememory/mv/AbstractPageMemoryMvPartitionStorage.java

```diff
@@ -931,4 +933,48 @@ public void close() {
             // No-op.
         }
     }
+
+    private int readHashIndexInlineSize(long metaPageId) throws IgniteInternalCheckedException {
```

Comment:
Page handlers can be used instead. I believe you should use them when you move these methods to a proper place

Reply:
I'll try to move it inside the tree.

Reply:
Fix it